src/polyfills.js

```diff
@@ -0,0 +1,18 @@
+import promisePoly from 'core-js/fn/promise';
+import 'whatwg-fetch/fetch.js';
+import 'Base64/base64.js';
+
+let scope;
+
+// check if we're running in an eco-system that uses window
+// if it doesnt exist use global
+if (typeof window !== 'undefined') {
+  scope = window;
+} else {
+  scope = global;
+}
```

Comment:
Use the existing global module, or remove it if it's only going to be used in this one place now.